examples/project_references/lib_b/BUILD.bazel

```diff
@@ -27,13 +26,13 @@ ts_project(
     ],
 )
 
-jasmine_test(
-    name = "test",
-    args = ["*.spec.js"],
-    chdir = package_name(),
-    data = [
-        "index.spec.js",
-        ":lib_b",
-    ],
-    node_modules = "//examples:node_modules",
-)
+# jasmine_test(
```

Comment:
What was the issue here?

Reply:
We need to bump the rules_jasmine dep to be compatible with rules_js 2.0. It will be easier to just remote the akward dep between rules_ts and rules_jasmine and use a straight js_test instead.

Reply:
✅